src/shell/store/content.js

```diff
@@ -275,16 +275,24 @@ export function searchItems(
       type: "FETCH_RESOURCE",
       uri: `${CONFIG.API_INSTANCE}/search/items?q=${term}&order=${query.order}&dir=${query.dir}&limit=${query.limit}`,
       handler: res => {
-        if (Array.isArray(res.data)) {
-          dispatch({
-            type: "SEARCH_ITEMS_SUCCESS",
-            data: res.data.reduce((acc, item) => {
-              acc[item.meta.ZUID] = item;
-              return acc;
-            }, {})
-          });
+        if (res.status === 200) {
+          if (Array.isArray(res.data)) {
```

Comment:
Combine the conditions of line 278 & 279. Otherwise we would swallow an issue of a 200 response which returned an unexpected payload.